main/saturn-node.js

```diff
@@ -124,6 +126,7 @@ async function start (/** @type {Context} */ ctx) {
         console.log('Saturn node is up and ready (API URL: %s)', apiUrl)
         webUrl = `${apiUrl}webui`
         ready = true
+        moduleExitReason = null
```

Comment:
Actually, we need to clean `moduleExitReason` at the time when we start the process. If the process does not start successfully (never prints the `API:` line), then your implementation never cleans `moduleExitReason`.

On second thought, maybe we don't need to clear this flag since it should always be set from the `exit` event listener.

Reply:
Good call, I went with your suggestion to clear it more reliably